### DISCUSSION TITLE
Displaying BLTouch LCD Voltage Menu

### DISCUSSION THREAD

**Discussion by fuzelet** [mriscoc/Ender3V2S1#751] [Q&A] [answered]
I was messing around with some BLTouch settings and am wondering if I am missing something to allow me to see the BLTouch Voltage menu.  I enabled "BLTOUCH_LCD_VOLTAGE_MENU" in "CONFIGURATION_ADV.H", recompiled and flashed the firmware, however no new menu appeared in any section of the LCD Screen. I did not receive any errors on compile either.  I tried looking for any documentation to see what this MENU presented to the user, however I was not able to find anything. Does anyone know if there is another setting that needs to be enabled to show this menu?  Is this menu supported in ProUI?

https://marlinfw.org/docs/features/lcd_menu.html
I do not even see this menu being listed here?

**Comment by mriscoc** [comment, marked as answer]
That menu is only compatible with Marlin UI, it is not enabled in ProUI
